fastai/torch_core.py

```diff
@@ -30,7 +30,15 @@
 
 # Cell
 @delegates(plt.subplots, keep=True)
-def subplots(nrows=1, ncols=1, figsize=None, imsize=3,suptitle=None, **kwargs):
+def subplots(
+    nrows: int=1,        # Number of rows in returned axes grid
+    ncols: int=1,        # Number of columns in returned axes grid
+    figsize: tuple=None, # Width, height in inches of the returned figure
+    imsize: int=3,       # Size (in inches) of images that will be displayed in the returned figure
+    suptitle=None,       # Title to be set to reutrned figure
+    **kwargs
```

Comment:
Formatting nit, just add one space before each docment, and no types. Please see the [style guide](https://github.com/muellerzr/fastai-docment-sprint/blob/master/docs_src/style-guide.md)

Reply:
Thank you. I will fix those issues.

Reply:
@muellerzr Why are we removing types? Or did you mean to state "and no space before types"?

Reply:
That's what I meant to say, sorry!

Reply:
@muellerzr ok I will have types, but no space before them. This will give me practice with going back to an older commit